torchtext/models/__init__.py

```diff
@@ -0,0 +1,7 @@
+from .multiheadattention import MultiheadInProject, MultiheadOutProject, \
```

Comment:
In general, I think those abstractions should live in a `ops` or `modules` folder, instead of `models`, as those are not really models but model building blocks